### PR TITLE
common: properties parsing mechanism is fixed

### PR DESCRIPTION
**Pull request by rkhozinov**

### CODE REVIEW

File: servers/partner-ci/common.yaml [hunk 1]
```diff
@@ -290,8 +290,10 @@
     builders:
       - shell: |
           #!/bin/bash
+          IFS=';'
           echo > common.properties
           if [ ! -z "${COMMON_PROPS}" ]; then
+
```
Comment (naumvd95) [inline servers/partner-ci/common.yaml]: space

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/common.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/common.yaml]: ok

**Review by naumvd95 — DISMISSED** [review]

File: servers/partner-ci/9.x/nsxv.yaml [hunk 2]
```diff
@@ -150,6 +150,7 @@
 
       - '{custom-name}.qa':
           publish_results: false
+          scm_wipe_workspace: true
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/nsxv.yaml]: `scm_wipe_workspace` is the global parameter and its default value is **true**
global variables are passed to all jobs as is.
so we don't reassign the same value

Comment (rkhozinov) [reply servers/partner-ci/9.x/nsxv.yaml]: + the same for the 135 line

File: servers/partner-ci/common.yaml [hunk 3]
```diff
@@ -239,11 +239,11 @@
                   sudo cp /var/log/libvirt/libvirtd.log ${{WORKSPACE}}/libvirtd_total.log
                   sudo chown jenkins:jenkins ${{WORKSPACE}}/libvirtd_total.log
       - postbuildscript:
-          script-only-if-succeeded: True
-          script-only-if-failed: False
+          script-only-if-succeeded: False
+          script-only-if-failed: True
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml:243]: It's ok

File: servers/partner-ci/9.x/nsxv.yaml (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/9.x/nsxv.yaml]: .

**Review by rkhozinov — COMMENTED** [review]
it's ok for me

**Review by naumvd95 — APPROVED** [review]
awesome

**Pull request merged**